evcc.dist.yaml

```diff
@@ -252,20 +252,29 @@ eebus:
 messaging:
   events:
     start: # charge start event
-      title: Charge started
-      msg: Started charging in "${mode}" mode
+      title: Started charging of ${vehicleTitle} in ${mode} mode
+      msg: |
+        Start Time {{printf `%02d:%02d` now.Hour now.Minute}}
+        SoC {{round (.vehicleSoc) 0}}%
+        Range ${vehicleRange}km
     stop: # charge stop event
-      title: Charge finished
-      msg: Finished charging ${chargedEnergy:%.1fk}kWh in ${chargeDuration}.
+      title: Charge of ${vehicleTitle} finished
+      msg: |
+        End Time {{printf `%02d:%02d` now.Hour now.Minute}}
+        SoC {{round (.vehicleSoc) 0}}%
+        Range ${vehicleRange}km
+        Charged Energy ${chargedEnergy:%.1fk}kWh
+        Total Price {{round (.sessionPrice) 2}}€
```

Comment:
s/Price/Cost/

Reply:
Das funktioniert auch nur wenns ein "echter" Tarif ist, aber nciht mit Co2

Reply:
> s/Price/Cost/

Meinst Du anstatt Total Price nur Price oder Cost?

Reply:
> s/Price/Cost/


> Das funktioniert auch nur wenns ein "echter" Tarif ist, aber nciht mit Co2

Zeile raus oder auskommentieren?

Reply:
Schau gerne mal ob Du das in irgendeiner Variable findest, sonst müsste es raus.

> Meinst Du anstatt Total Price nur Price oder Cost?

Total Cost wäre auch fein.